.circleci/config.yml

```diff
@@ -29,6 +29,12 @@ jobs:
       - run:
           name: Run Specs
           command: bundle exec rake docker:spec
+      - run:
+          name: Run Specs (Fedora 5 compatibility)
+          command: bundle exec rake docker:spec
+          environment:
+            FCREPO_VERSION: 5.0.0-RC-1
```

Comment:
@mbklein Is there an easy way to configure the version of Fedora that the Docker container uses?

Reply:
Looking into it, but first I have to get a Fedora 5 container built.